core/controllers/editor.py

```diff
@@ -739,6 +687,7 @@ class StateYamlHandler(EditorHandler):
     layer.
     """
 
+    @acl_decorators.open_access
```

Comment:
It is called from `VersionDiffVisualizationDirective.js`

Yes, same authorization as can_play_exploration will work but it doesn't take exploration id as argument (which is required for can_play_exploration). Should I just implement a new decorator `can_get_yaml_from_states` or not ?